jiraworklog/src/Container/Login/Login.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { Button, Grid, Header, Segment, Message } from 'semantic-ui-react'
+import { Form, Input } from 'semantic-ui-react-form-validator';
+import axios from 'axios';
+// import inputHoc from '../../Hoc/input';
+import 'semantic-ui-css/semantic.min.css';
+import base64 from 'base-64';
+
+class Login extends Component {
+state = {
+  email: "",
+  apitoken: "",
+  apiurl: "",
+  errorMsg: false
+}
+
+  fetchData = async () => {
+    const { email, apitoken, apiurl } = this.state;
+    const headers = { Authorization: 'Basic ' + base64.encode(`${email}:${apitoken}`), 'Content-Type': 'application/json', Accept: 'application/json' }
+
+    try {
+      const response = await axios.get(`${apiurl}/rest/api/2/project`, { headers });
```

Comment:
I have asked you to create separate utility file to manage api requests and reuse that functionality, why this is not done?